engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/formatter/ConfluentAvroMessageFormatter.scala

```diff
@@ -23,24 +29,59 @@ private[confluent] class ConfluentAvroMessageFormatter(schemaRegistryClient: Sch
 
   schemaRegistry = schemaRegistryClient
 
-  def writeTo(data: Array[Byte], output: PrintStream): Unit = {
-    val obj = deserialize(data)
-    val schema = AvroSchemaUtils.getSchema(obj)
+  def genericToJson(obj: AnyRef, schema: Schema): Json = {
+    toJson(obj, schema, new GenericDatumWriter[AnyRef](schema))
```

Comment:
I know that it was before this way, but imo we should use `AvroAvroUtils.xxxData.genericDatumWriter` instead of using constructor directly. We have some fixes in our `AvroAvroUtils` regards to logical type handling, `GenericRecordWithSchemaId`, `StringForcing` and so on and would be nice to keep it consistent

Reply:
Good point, it is moved to separate PR.